cli/src/cmd/forge/test.rs

```diff
@@ -438,29 +438,45 @@ fn short_test_result(name: &str, result: &TestResult) {
     let status = if result.success {
         Paint::green("[PASS]".to_string())
     } else {
-        let txt = match (&result.reason, &result.counterexample) {
-            (Some(ref reason), Some(ref counterexample)) => {
+        let txt = match (&result.reason, &result.counterexample, &result.counterexample_sequence) {
+            (Some(ref reason), Some(ref counterexample), None) => {
                 format!("[FAIL. Reason: {reason}. Counterexample: {counterexample}]")
             }
-            (None, Some(ref counterexample)) => {
+            (None, Some(ref counterexample), None) => {
                 format!("[FAIL. Counterexample: {counterexample}]")
             }
-            (Some(ref reason), None) => {
+            (Some(ref reason), None, None) => {
                 format!("[FAIL. Reason: {reason}]")
             }
-            (None, None) => "[FAIL]".to_string(),
+            (Some(ref reason), None, Some(sequence)) => {
+                let mut inner_txt = "".to_string();
+
+                for checkpoint in sequence {
+                    inner_txt += format!("{checkpoint}\n").as_str();
+                }
+
+                format!("###\nFAIL. Reason: {reason}. | Sequence:\n{inner_txt}\n")
+            }
+            _ => "[FAIL. Reason:".to_string(),
         };
 
         Paint::red(txt)
     };
 
-    println!("{} {} {}", status, name, result.kind.gas_used());
+    println!("{} {} {}", status, name, result.kind.report());
 }
 
 pub fn custom_run(args: TestArgs, include_fuzz_tests: bool) -> eyre::Result<TestOutcome> {
     // Merge all configs
     let (config, mut evm_opts) = args.config_and_evm_opts()?;
 
+    let mut test_options = TestOptions {
+        include_fuzz_tests,
+        invariant_depth: config.invariant_depth,
+        invariant_fail_on_revert: config.invariant_fail_on_revert,
+        invariant_call_override: config.invariant_call_override,
+    };
```

Comment:
nit: At this point we should probably construct `TestOptions` outside of `custom_run` and have `custom_run` take a `TestOptions`